scripts/extract-monorepo-docs.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -
+
+MONOREPO_CODELOAD_URL='https://codeload.github.com/storybookjs/storybook/tar.gz'
+BRANCH=${1:-6.0-docs}
```

Comment:
What does this do? 🤔

Reply:
Uses the first CLI argument, defaulting to `6.0-docs`.